packages/react/src/header/Header.tsx

```diff
@@ -5,14 +5,22 @@ import { HeaderBrand } from './HeaderBrand';
 import { HeaderContainer } from './HeaderContainer';
 
 export type HeaderProps = {
```

Comment:
Added prop comments and sorted in a way that seemed logical - I haven't removed any props.